doc/internal/man3/evp_keymgmt_util_export_to_provider.pod

```diff
@@ -65,6 +67,10 @@ evp_keymgmt_util_fromdata() can be used to add key object data to a
 given key I<target> via a B<EVP_KEYMGMT> interface.  This is used as a
 helper for L<EVP_PKEY_fromdata(3)>.
 
+In all functions that take a I<selection> argument, the selection is used to
+constraint the information requested on export. It is also used in the cache
+so that only the same selection is matched when looking for key data exports.
```

Comment:
It would be then `only the same or superset selection is matched...`

Reply:
If we change the code, i would say: "It is also used in the cache so that key data is guaranteed to contain all the information requested in the selection."

Reply:
done